Tests/Admin/AdminHelperTest.php

```diff
@@ -18,6 +18,9 @@
 
 class AdminHelperTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * @covers Sonata\AdminBundle\Admin\AdminHelper::getChildFormBuilder
+     */
```

Comment:
the background of this change is, that i would like to know which parts are not directly covered by a test. Creating an Admin in a test does not mean it has a test and i want to find out which parts needs to be tested "better" otherwise code coverage would say, yes this is covered!

anyway, we currently have some `@covers` annotations in our project, for example in `AdminTest`